### DISCUSSION TITLE
Disable/Delete Zen tests

### DISCUSSION THREAD

**Discussion by itaysharir** [monkeytypegame/monkeytype#4290] [Ideas] [answered]
I wanted to know what Zen mode was, so i've done a zen test and got like 200 wpm, which messed up my entire profile. For example: My best score and the graph that shows your typing speed over time.

**Comment by samiulmuztaba** [comment]
Same

**Comment by Miodec** [comment, marked as answer]
You can filter them out using the account page filters.

**Comment by LucasV727** [comment]
well it just happened to me I wanted to see what zen was so I just typed in random string then ctrl + enter and it just submitted it and now there is a huge spike that I want to remove TwT

**Reply by fehmer** [reply]
https://github.com/monkeytypegame/monkeytype/discussions/4290#discussioncomment-9212718
